nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -137,6 +137,10 @@ void SmacPlannerHybrid::configure(
     node, name + ".viz_expansions", rclcpp::ParameterValue(false));
```

Comment:
Remove this parameter + update the other expansions visualization publisher / activation statements to use `debug_visualizations` too.